packages/core/test/endowmentsToolkit.spec.js

```diff
@@ -92,7 +92,43 @@ test('getEndowmentsForConfig - siblings', (t) => {
   }
 })
 
-test('getEndowmentsForConfig - getter', (t) => {
+test('getEndowmentsForConfig - knownWritableFields', (t) => {
+  const getEndowmentsForConfig = prepareTest()
+  const sourceGlobal = {
+    a: 1,
+    b: { c: 2 },
+    d: 3,
+  }
+  const config = {
+    globals: {
+      a: true,
+      'b.c': true,
```

Comment:
TODO: check what happens if 
```
'b.c':true,
'b.c.x':true
```

- I think the current implementation will fail
- I think copyValueAtPath might also not be handling that correctly.

Can be fixed or forbidden

Reply:
done